config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml

```diff
@@ -28,7 +28,6 @@ spec:
          hostName: edpm-compute-0
          ansible:
            ansibleHost: 192.168.122.100
-           ansibleUser: root
```

Comment:
There's no default set for ansibleUser. Shouldn't it be cloud-admin?

Reply:
I'm adding cloud-admin on install_yamls: https://github.com/openstack-k8s-operators/install_yamls/pull/531

Reply:
As long as it's set here in the samples as well. thanks.